core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -721,16 +732,15 @@ class SparkContext(config: SparkConf) extends Logging {
   }
 
   /**
-   * Creates a new RDD[Long] containing elements from `start` to `end`(exclusive), increased by
+   * Creates a new `RDD[Long]` containing elements from `start` to `end`(exclusive), increased by
    * `step` every element.
    *
    * @note if we need to cache this RDD, we should make sure each partition does not exceed limit.
-   *
    * @param start the start value.
    * @param end the end value.
    * @param step the incremental step
    * @param numSlices the partition number of the new RDD.
-   * @return
+   * @return RDD containing distributed range
```

Comment:
Nit: RDDs are really pointers and bookeeping. The "represent" rather than "contain"